pages/common/node.md

```diff
@@ -15,9 +15,9 @@
 
 `node -e "{{code}}"`
 
-- Evaluate and print result, useful to see node's dependencies versions:
+- Evaluate and print the result, this specific command is useful to print node's dependencies versions:
 
-`node -p "{{process.versions}}"`
+`node -p "process.versions"`
```

Comment:
@navarroaxel can you check this line, Is this allowed? (Related PR #8898)

Reply:
I'm ok with this change because we're talking about a specific value for the `-p` argument.